inc/simplepdf.class.php

```diff
@@ -120,17 +118,28 @@ public function setHeader($msg)
         $this->header = $msg;
         $this->pdf->resetHeaderTemplate();
         $this->pdf->SetTitle($msg);
-        $configurationValues = Config::getConfigurationValues('core', ['version']);
-        $current_version     = $configurationValues['version'];
-        switch ($current_version) {
-            case '0.85.3':
-            case '0.85.4':
-            case '0.85.5':
-                $this->pdf->SetHeaderData('fd_logo.jpg', 15, $msg, '');
-                break;
+        $config = PluginPdfConfig::getInstance();
+        if (
+            Plugin::isPluginActive('branding')
+            && $config->getField('use_branding_logo')
+        ) {
+            $brandingUtils = new \PluginBrandingUtils();
+            $logo = $brandingUtils->getFilePath(Session::getActiveEntity(), 'main_logo');
+            $this->pdf->SetHeaderData($logo, 15, $msg, '');
+        } else {
+            $configurationValues = Config::getConfigurationValues('core', ['version']);
+            $current_version     = $configurationValues['version'];
+            $path = Plugin::getPhpDir('pdf') . '/pics/';
+            switch ($current_version) {
+                case '0.85.3':
+                case '0.85.4':
+                case '0.85.5':
+                    $this->pdf->SetHeaderData($path . 'fd_logo.jpg', 15, $msg, '');
+                    break;
```

Comment:
`setup.php` define this

`define('PLUGIN_PDF_MIN_GLPI', '10.0.0');`

So we can drop support of old GLPI version for `fd_logo.jpg` (instead of `fd_logo.png` (use now))